### PR TITLE
improve the function of simple_test_bboxes

### PR DESCRIPTION
**Pull request by yuzhj**

### CODE REVIEW

**Comment by yuzhj** [comment]
@hellock @yhcao6 I made some new changes to make the code more reasonable

**Comment by hellock** [comment]
There are conflicts.

**Comment by yuzhj** [comment]
> There are conflicts.

Thanks, I have finished conflicts now

File: mmdet/models/roi_heads/test_mixins.py [hunk 1]
```diff
@@ -72,11 +72,18 @@ def simple_test_bboxes(self,
...
-                bbox_pred, ) if bbox_pred is not None else [None, None]
+            if bbox_pred is not None:
+                bucket_cls_preds, bucket_offset_preds = bbox_pred
+                bucket_cls_preds = bucket_cls_preds.split(
+                    num_proposals_per_img, 0)
+                bucket_offset_preds = bucket_offset_preds.split(
+                    num_proposals_per_img, 0)
+                bbox_pred = tuple(zip(bucket_cls_preds, bucket_offset_preds))
+            else:
+                bbox_pred = (None, ) * len(proposals)
```
Comment (yhcao6) [inline mmdet/models/roi_heads/test_mixins.py:86]: If the bbox prediction of another detector other than SABL is tuple, then we split it as `bucket_cls_preds, bucket_offset_preds = bbox_pred` may be stuck. So move the code sniper to sabl_head.py maybe a better choice.

Comment (yuzhj) [reply mmdet/models/roi_heads/test_mixins.py]: @yhcao6 Thanks for your suggestion, I have finished the modification now, look forward to a further review, thanks

**Review by yhcao6 — APPROVED** [review]

File: mmdet/models/roi_heads/test_mixins.py (hunk 1 above)
Comment (yhcao6) [reply mmdet/models/roi_heads/test_mixins.py]: Thanks for your contribution, approved

**Pull request merged**